trade-finance-manager-ui/templates/case/amendments/amendment-add-banks-decision-check-answers.njk

```diff
@@ -25,9 +25,8 @@
       <a
         class="govuk-link"
         href="/case/{{ amendment.dealId }}/facility/{{ amendment.facilityId }}/amendment/{{ amendment.amendmentId }}/banks-decision"
-        aria-label="Change banks decision"
         data-cy="amendment--change-bank-decision-link">
-        Change
+        Change <span class="govuk-visually-hidden">Bank's decision</span>
```

Comment:
Should this be `bank's decision` instead of `Bank's decision`?

Reply:
Thanks Luke, good notice, changed.